### PR TITLE
qemu_vm: includes support for mem policy and host-nodes

### PR DESCRIPTION
**Pull request by mcasquer**
qemu_vm: includes support for mem policy and host-nodes

Includes the support of the policy and host-nodes options
for the machine memory backend.

ID: 1398
Signed-off-by: mcasquer <mcasquer@redhat.com>

### CODE REVIEW

**Comment by mcasquer** [comment]
@yanan-fu could you review this PR? Thanks !

**Comment by yanan-fu** [comment]
This patch aim to add new options for the machine mem object, so it can bind the machine mem to the specified host numa node.
@YongxueHong @luckyh Could you please help  review ?

File: virttest/qemu_vm.py [hunk 1]
```diff
@@ -1097,6 +1097,10 @@ def add_memorys(devices, params):
                 name = "machine_mem"
                 backend_options = {}
                 backend_options["size_mem"] = "%sM" % params["mem"]
+                if params.get("vm_mem_policy"):
+                    backend_options["policy_mem"] = params.get("vm_mem_policy")
+                if params.get("vm_mem_host-nodes"):
+                    backend_options["host-nodes"] = params.get("vm_mem_host-nodes")
```
[the remark this answers is not in the document]
Comment (mcasquer) [reply virttest/qemu_vm.py:1103]: @YongxueHong I understand line 1102 needs to be `vm_mem_host_nodes` as well

[the remark this answers is not in the document]
Comment (mcasquer) [reply virttest/qemu_vm.py]: Could you review again this PR? Thanks!

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Review by yanan-fu — APPROVED** [review]
Ack

**Comment by mcasquer** [comment]
It seems this PR could be merged :grinning:

**Pull request merged**